### PR TITLE
Add news for 0.26.0, 0.25.1, 0.24.1

### PR DESCRIPTION
**Pull request by ajeddeloh**
Add news for branched releases
Add news for 0.26.0

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,15 @@
+11-June-2018 IGNITION v0.26.0
+
+  Features:
+
+    - Expand partitioning support to support filesystem reuse like semantics
```
Comment (bgilbert) [inline NEWS]: Rather than comparing the feature to a different feature, can we just say what it does?

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+    - Fail blackbox tests when Ignition encounters critical errors
+
+  Bug Fixes:
+
+    - Fix an issue in timeout logic causing http(s) requests to sometimes fail
+    - Do not log non-critial errors with CRITIAL log level
```
Comment (bgilbert) [inline NEWS]: `critical`, `CRITICAL`

**Comment by arithx** [comment]
s,0.26.0,v0.26.0,g

**Comment by ajeddeloh** [comment]
Fixed typos, added `v`, redescribed partitioning changes

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,28 @@
+11-June-2018 IGNITION v0.26.0
+
+  Features:
+
+    - Expand partitioning support to support partition matching, specifying
```
Comment (dgonyeo) [inline NEWS]: nit: `support to support` sounds weird

File: NEWS [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+      partitions should not exist, and creating existing partitions
+    - Fail blackbox tests when Ignition encounters critical errors
```
Comment (dgonyeo) [inline NEWS]: nit: `encounters critical-level logs` might be a tad clearer

File: NEWS [hunk 5]
```diff
@@ -1,3 +1,28 @@
+11-June-2018 IGNITION v0.26.0
+
+  Features:
+
+    - Expand partitioning to support partition matching, specifying
+      partitions should not exist, and creating existing partitions
```
Comment (bgilbert) [inline NEWS]: `specifying that a partition should not exist`

Could also drop `Expand partitioning to`.

`creating existing partitions` reads oddly; `creating partitions that may already exist`?

Comment (ajeddeloh) [reply NEWS]: whoops, was supposed to be "recreating existing partitions"

**Pull request merged**